WPFMainApp/App.cs

```diff
@@ -85,4 +89,20 @@ private static void EnsureConfiguration(string configurationFilePath, string con
             }, new JsonSerializerOptions { WriteIndented = true });
         }
     }
+
+    public void End(Browser selectedBrowser)
```

Comment:
Not a fortunate name for the method. Do not call `Shutdown` here, just save settings to file.

Reply:
Resolved in the recent commit